relay-dynamic-config/src/defaults.rs

```diff
@@ -122,12 +131,16 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
         | is_resource.clone()
         | is_mobile.clone()
         | is_interaction
-        | is_http.clone())
+        | is_http.clone()
+        | is_cache.clone())
         & duration_condition.clone();
 
-    let know_modules_condition =
-        (is_db.clone() | is_resource.clone() | is_mobile.clone() | is_http.clone())
-            & duration_condition.clone();
+    let know_modules_condition = (is_db.clone()
+        | is_resource.clone()
+        | is_mobile.clone()
+        | is_http.clone()
+        | is_cache.clone())
```

Comment:
By adding `is_cache` to `known_modules_condition`, we get multiple tags that we might not need (see `.when(know_modules_condition.clone()),` below). Could we start by only adding the `cache.hit` tag instead?

Reply:
Yeah we can, works for me!

Reply:
done!